static/js/dashboard_routes.js

```diff
@@ -0,0 +1,99 @@
+import App from './containers/App';
+
+const errorLoading = error => {
+  throw new Error(`Dynamic page loading failed: ${error}`);
+};
+
+const loadRoute = cb => (
+  module => cb(null, module.default)
+);
+
+const username = SETTINGS.user ? SETTINGS.user.username : '';
+
+export const routes = {
```

Comment:
Unfortunately no, the calls to `import` have to be statically known to webpack before any of the code runs in order to figure out where the module boundaries should be (and to then do the splitting). We can't use this to derive the test routes from because the test routes do not run in a webpack environment, so when `nodejs` tries to parse the files it will complain about non-top level import statements. It's messy I know :(